Makefile

```diff
@@ -3,10 +3,10 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 3.11.1
+VERSION ?= $(shell cat VERSION)
```

Comment:
Can we add a default? `:=3.11.1`

Reply:
ohohohoho nono it is good I just started review I couldn't see that

Reply:
can you write readme `docs/upgrading-gatekeeper.md`

Reply:
There's a `release.md` doc that was also updated in this PR.